### ISSUE TITLE
return timestamp with timezone info included

### ISSUE THREAD

**Issue by newgene** [biothings/biothings.api#78]
The current timestamps returned from metadata does not includes timezone info, for example, the `build_date` from http://mygene.info/metadata:

```
"build_date": "2020-07-19T04:00:00.016211",
```

This is the result of Python's `isoformat` method:

```
In [6]: datetime.datetime.now().isoformat()
Out [6]: '2020-07-27T13:25:08.535614
```

By switching to use this:
```
In [7]: datetime.now(timezone.utc).astimezone().isoformat()
Out[7]: '2020-07-27T13:25:08.535614-07:00'
```

the output timestamp will include the timezone info.

Ref: https://stackoverflow.com/questions/3401428/how-to-get-an-isoformat-datetime-string-including-the-default-timezone

**Comment by newgene** [comment]
implemented in 91516b76 f6e79b5b 4cd8ad11.

**Comment by newgene** [comment]
also this commit: 00416a72dc09bbfa20933d0498f7b0441d9a5032
